dipy/utils/testing.py

```diff
@@ -0,0 +1,173 @@
+from pytest import approx
+import sys
+import os
+from numpy.core import around, number, float_, result_type, array
+from numpy.core.numerictypes import issubdtype
+from numpy.core.fromnumeric import any as npany
+import numpy as np
+
+
+def assert_equal(value1, value2):
+	if not 'numpy' in str(type(value1)):
+		value1 = np.array(value1)
+	if not 'numpy' in str(type(value2)):
+		value2 = np.array(value2)
+	try:
+		assert np.all(value1 == value2)
```

Comment:
why are you using `np.all`? Not needed I think

Reply:
as you can see [here](https://github.com/pytest-dev/pytest/pull/2492/files#diff-a4d4d6cc8ecc32438c8a4da391852f22R325),  `pytest` manage numpy array and list of numpy array, so I still think that `np.all` is not needed